docs/military/williamtown.md

```diff
@@ -4,4 +4,115 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Williamtown Approach    | Willy Approach   | 135.700         | WLM_APP                                   |
+| Williamtown ADC    | Willy Tower  | 118.300         | WLM_TWR        |
+| Williamtown SMC    | Willy Ground  | 121.800         | WLM_GND        |
+| Williamtown ACD    | Willy Delivery  | 130.350         | WLM_DEL       |
+| Williamtown ATIS    | Williamtown ATIS  | 134.450         | YWLM_ATIS       |
```

Comment:
Can remove the callsign for the ATIS (other ADs have an empty cell for the ATIS callsign from memory)

Reply:
Ty Tyler :)